class/List.php

```diff
@@ -16,11 +16,11 @@ function prep_favorites()
   function prep_data($row)
   {
     global $Parse;
-  
+
```

Comment:
Note: in general, I'm trying not to include whitespace changes in my PRs, but sometimes they get caught up in a section of a commit, despite adding `git add -p`.